GoogleForCreators/web-stories-wp
# Story Editor: Added Cover Text Sets

## Summary
Adds text sets under `Cover` category in this design file:
https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=837%3A3585

## User-facing changes
NA

## Testing Instructions
Enable the text sets flag in experiments and see the new text sets in the story editor.

## Screenshots
<img width="348" alt="Screen Shot 2020-09-16 at 2 08 37 PM" src="https://user-images.githubusercontent.com/35983235/93388002-4d267380-f827-11ea-9614-f0044a915408.png">


---



Partial for #4078
